lib/web/apiserver_test.go

```diff
@@ -2619,6 +2620,104 @@ func TestCheckAccessToRegisteredResource(t *testing.T) {
 	}
 }
 
+func TestAuthExport(t *testing.T) {
+	ctx := context.Background()
+
+	env := newWebPack(t, 1)
+	clusterName := env.server.ClusterName()
+
+	proxy := env.proxies[0]
+	pack := proxy.authPack(t, "test-user@example.com", nil)
+
+	for _, tt := range []struct {
+		name           string
+		authType       string
+		expectedStatus int
+		assertBody     func([]byte)
+	}{
+		{
+			name:           "all",
+			authType:       "",
+			expectedStatus: http.StatusOK,
```

Comment:
Could we do some format validation in the response? Ie, check that it is valid PEM/DER/authorized_key/etc ?

Reply:
Yes
I added a couple of checks (correct format and some values)